modules/data-streams/src/main/java/org/elasticsearch/datastreams/lifecycle/DataStreamLifecycleService.java

```diff
@@ -296,6 +300,25 @@ public void triggered(SchedulerEngine.Event event) {
                     event.getTriggeredTime()
                 );
                 run(clusterService.state());
+                dataStreamLifecycleErrorsPublisher.publishDslErrorEntries(new ActionListener<AcknowledgedResponse>() {
+                    @Override
+                    public void onResponse(AcknowledgedResponse acknowledgedResponse) {
+                        assert acknowledgedResponse.isAcknowledged() : "updating the health info is always acknowledged";
+                    }
+
+                    @Override
+                    public void onFailure(Exception e) {
+                        logger.trace(
```

Comment:
Nit: Since this is logging an error and it only occurs once every 5 minutes, what do you think about setting it to debug level?